CONTRIBUTING.md

```diff
@@ -39,6 +39,8 @@ Qt:
 * Provide **screenshots** for UI related changes.
 * Keep your git commit history **clean** and **precise**. Commits like
   `xxx fixup` should not appear.
+* Commit message should state not only what has been changed, but also why
```

Comment:
Well, there's nothing else out there that would say that commit message should say what is being changed – above there is a sentence about PR message, but not really about commits..